edifact-edi/2.2/modules/ROOT/pages/edifact-edi-connector-config-topics.adoc

```diff
@@ -68,38 +70,38 @@ These control the parser operation and the types of error conditions that reject
 
 [%header%autowidth.spread]
 |===
-|XML Value |Studio Option
-|enforceLengthLimits="true" |Enforce the minimum and maximum lengths for receive values
-|enforceValueRepeats="true" |Enforce the repetition count limits for receive values
-|enforceReadCharacters="true" |Enforce valid characters for read. This only applies when using syntax levels UNOA and UNOB
-|allowUnknownSegments="false" |Allow unknown segments in a message
-|enforceSegmentOrder="true" |Enforce segment order in a message
-|allowUnusedSegments="false" |Allow segments marked as `Unused` in a message
-|enforceSegmentRepeats="true" |Enforce segment repetition count limits in a message
-|receiveEncodingOverride="false" |Java character encoding used for Receive messages. This overrides receive UNB
-|requireUniqueInterchanges="true" |Require unique Interchange Control References (UNB.5). Records the previously processed interchange numbers and rejects any duplicate interchange numbers from the same partner. This is determined by the interchange sender and receiver identification. If set to false, it processes the received interchange and leaves it to the application flow to track numbers and handle appropriately.
-|requireUniqueMessages="false" |Enforce globally unique Message Reference Numbers (UNH.1) for received messages. By default, message reference numbers only need to be unique with a particular interchange. If set to true, this requires message reference numbers to be unique across all interchanges received from the same partner and application. This is determined by the interchange sender and recipient identification.
+|XML Value (default) |Studio Option
+|enforceLengthLimits="true" |Enforce the minimum and maximum lengths for receive values.
+|enforceValueRepeats="true" |Enforce the repetition count limits for receive values.
+|enforceReadCharacters="true" |Enforce valid characters for a Read. Only applies when using syntax levels `UNOA` and `UNOB`.
+|allowUnknownSegments="false" |Allow unknown segments in a message.
+|enforceSegmentOrder="true" |Enforce the segment order in a message.
+|allowUnusedSegments="false" |Allow segments marked as `Unused` in a message.
+|enforceSegmentRepeats="true" |Enforce segment repetition count limits in a message.
+|receiveEncodingOverride="false" |Java character encoding used for Receive messages. This overrides receive UNB.
+|requireUniqueInterchanges="true" |Require unique Interchange Control References (such as UNB.5). The default behavior records the interchange numbers previously processed and rejects duplicate interchange numbers from the same partner. It is derived from the interchange Sender and Receiver identification. Setting to `false` enables received interchange processing to continue and relies on the application flow to track the numbers and handle them appropriately.
```

Comment:
```suggestion
|requireUniqueInterchanges="true" |Require unique Interchange Control References (such as UNB.5). The default behavior records the interchange numbers previously processed and rejects duplicate interchange numbers from the same partner. It is derived from the interchange Sender and Receiver identification. Setting to `false` enables received interchange processing to continue and relies on the application flow to track the numbers and manage them appropriately.
```

Reply:
Kim many times edited handle to manage. Feel free to take it or leave it

Reply:
Thanks!